go/fn/examples/example_logger_injector_test.go

```diff
@@ -26,7 +26,7 @@ func injectLogger(rl *fn.ResourceList) error {
 		return err
 	}
 	for i, obj := range rl.Items {
-		if obj.APIVersion() == "apps/v1" && (obj.Kind() == "Deployment" || obj.Kind() == "StatefulSet" || obj.Kind() == "DaemonSet" || obj.Kind() == "ReplicaSet") {
+		if obj.GetAPIVersion() == "apps/v1" && (obj.GetKind() == "Deployment" || obj.GetKind() == "StatefulSet" || obj.GetKind() == "DaemonSet" || obj.GetKind() == "ReplicaSet") {
```

Comment:
(Not this PR). I think syntax sugar for complete gvk comparison will be helpful.

Reply:
Yeah, I have another PR for GVK comparison functions (needed by set-namespace func). I'll have it open once this PR is merged.